samples/sensor/accel_polling/src/main.c

```diff
@@ -51,6 +51,30 @@ static int print_accels(const struct device *dev)
 	return 0;
 }
 
+static int set_sampling_freq(const struct device *dev)
+{
+	int ret;
+	struct sensor_value odr;
+
+	ret = sensor_attr_get(dev, SENSOR_CHAN_ACCEL_XYZ,
+			SENSOR_ATTR_SAMPLING_FREQUENCY, &odr);
+
```

Comment:
Would it make sense to return if device return `-ENOTSUP`?

Reply:
In a lot of drivers the "attr_get" function is not implemented, so if we return here the sample will not work for these sensors. My idea was that in the case we don't get an answer, we just assume that the frequency isn't correct, and try to set one. If the frequency was set, and we change it to another frequency than the default one, it's not really a problem, as we just need to fetch samples every second.